tests/integration/test_script_runner.py

```diff
@@ -187,6 +187,20 @@ class BackendDependencies(enum.Enum):
         "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples/first_3_files",
         "extra_backend_dependencies": BackendDependencies.SPARK,
     },
+    {
+        "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/database/mssql_yaml_example.py",
+        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+        "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples/first_3_files",
+        "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
+        "extra_backend_dependencies": BackendDependencies.MSSQL,
+    },
+    {
+        "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/database/mssql_python_example.py",
+        "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+        "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples/first_3_files",
+        "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
+        "extra_backend_dependencies": BackendDependencies.MSSQL,
```

Comment:
Thank you for this. I think the remaining piece is  adjusting `azure-pipelines-docs-integration.yml` 

1. under the stage : `docusaurus_tests`, `mssql: mssql` will need to be added under `services`
2. the step where we actually run the test script will need to be changed to specify `mssql` tests

`pytest -v --docs-tests -m docs --no-spark --mysql --mssql tests/integration/test_script_runner.py`

Reply:
that said `--no-spark` probably should be removed in a future PR.  The in-memory Spark tests need to be changed so they no longer import the `findspark` package, and work the same way as the filesystem Spark tests do

Reply:
> Thank you for this. I think the remaining piece is adjusting `azure-pipelines-docs-integration.yml`
> 
> 1. under the stage : `docusaurus_tests`, `mssql: mssql` will need to be added under `services`
> 2. the step where we actually run the test script will need to be changed to specify `mssql` tests
> 
> `pytest -v --docs-tests -m docs --no-spark --mysql --mssql tests/integration/test_script_runner.py

@Shinnnyshinshin Thank you for the guidance on my first docs PR! I expect that there will be more to come, so your efforts will reach beyond this PR.

Reply:
> that said `--no-spark` probably should be removed in a future PR. The in-memory Spark tests need to be changed so they no longer import the `findspark` package, and work the same way as the filesystem Spark tests do

@Shinnnyshinshin I will look into how this will get handled between pods.